.golangci.yaml

```diff
@@ -30,6 +31,15 @@ linters:
   disable:
     - errcheck
 
+linters-settings:
+  depguard:
+    list-type: blacklist
```

Comment:
woke barks at the use of blacklist here :(.

Reply:
annoying. Though.. I actually think this is the default for that field (I copy-pasted this setting from the example) so I think I can drop it. Let me try.

Reply:
Sneaky!

Reply:
ha, yeah, that worked 😎